src/core/components/wrapper/JsonAnimationWrapper.tsx

```diff
@@ -45,7 +45,7 @@ export class JsonAnimationWrapper extends BaseAnimationWrapper<JsonAnimationProp
         const transformArray = this._getTransformArray();
         const animations: ViewStyle[] = this._getViewStyleAnimationArray();
         return (
-            <Animated.View style={[{ transform: transformArray }, animations]}>
+            <Animated.View style={[{transform: transformArray}, animations, this.props?.animationWrapperStyles]}>
```

Comment:
We can add comment here that we are only doing this for `JsonAnimationWrapper.tsx`